xchange-binance/pom.xml

```diff
@@ -8,7 +8,8 @@
         <version>4.3.22-SNAPSHOT</version>
     </parent>
 
-    <artifactId>xchange-binance</artifactId>
+    <artifactId>xchange-binance-custom</artifactId>
+    <version>4.3.22.3</version>
```

Comment:
almost all of the maven changes in this PR are unacceptable

Reply:
fixed

Reply:
I still see strange repositories and custom artifacts in the poms